server/src/game/map/tile/city.ts

```diff
@@ -75,26 +85,8 @@ export class City {
     this.tiles.delete(coords);
   }
 
-  getUnits(): Unit[] {
-    return this.units;
-  }
-
-  getUnitPositions(): Coords[] {
-    return this.units.map(unit => unit.coords);
-  }
-
-  addUnit(unit: Unit): void {
-    this.units.push(unit);
-    if (this instanceof BarbarianCamp) {
-      unit.setBarbarian(true);
-    }
-  }
-
-  removeUnit(unit: Unit): void {
-    const unitIndex = this.units.indexOf(unit);
-    if (unitIndex > -1) {
-      this.units.splice(unitIndex, 1);
-    }
+  ownsUnit(unit: Unit): boolean {
```

Comment:
So, do Cities co-own all their Civ's Units currently in the city?

Reply:
Currently, cities co-own *all* of their Civ's units.